trin-core/src/lib.rs

```diff
@@ -1,6 +1,3 @@
-#[macro_use]
-extern crate lazy_static;
```

Comment:
If this is the only use, I guess we need to remove the `lazy_static` dependency from `trin-core/Cargo.toml`